src/Controls/src/Core/Compatibility/iOS/Extensions/ToolbarItemExtensions.cs

```diff
@@ -112,7 +112,9 @@ void UpdateIconAndStyle(ToolbarItem item)
 					}
 					item.IconImageSource.LoadImage(mauiContext, result =>
 					{
-						Image = result?.Value;
+						Image = item.IconImageSource is not FontImageSource
+							? result?.Value.ImageWithRenderingMode(UIImageRenderingMode.AlwaysOriginal)
```

Comment:
Can we add the color to the "ToUIBarButtonItem" call?

Or maybe we can add a bool that will indicate whether to toggle between the different rendering modes based on if BarTextColor is set?

That way the behavior should be pretty much the same as before